pkg/kubectl/cmd/util/factory.go

```diff
@@ -637,8 +637,8 @@ func NewFactory(optionalClientConfig clientcmd.ClientConfig) *Factory {
 				if t.Spec.Paused {
 					return true, nil
 				}
-				t.Spec.Paused = true
-				_, err := c.Extensions().Deployments(t.Namespace).Update(t)
+				body := []byte("{\"spec\":{\"paused\":true}}")
```

Comment:
Agree, here is a better version of this: https://github.com/openshift/origin/pull/10564/commits/bd484740f38b5790bb87a5d11bcbc406a39ac22a I will send the patch tomorrow morning if you agree. I'm find with the client functions.